packages/at_commons/lib/src/verb/abstract_verb_builder.dart

```diff
@@ -24,7 +24,10 @@ abstract class AbstractVerbBuilder implements VerbBuilder {
     } else if (atKeyObj.sharedWith.isNotNull) {
       key += '${VerbUtil.formatAtSign(atKeyObj.sharedWith)}:';
     }
-    key += '${atKeyObj.key!}${VerbUtil.formatAtSign(atKeyObj.sharedBy)}';
+    key += atKeyObj.key!;
+    if (atKeyObj.sharedBy.isNotNull) {
```

Comment:
In what situations can we have sharedBy == null?

Reply:
These changes are removed.